extensions/interactions/ImageClickInput/directives/image_click_input_interaction_directive.html

```diff
@@ -4,6 +4,13 @@
     padding: 8px;
     position: relative;
   }
+  .loading-indicator-gif {
+    display: block;
+    height: 120px;
+    margin-left: auto;
+    margin-right: auto;
+    width: 120px;
+  }
```

Comment:
This style is used in multiple places just move it to *oppia.css*

Reply:
Also, what if an image is smaller than 120 px width/height?

Reply:
Yeah that maybe a problem here. But we need to keep a fixed size of the indicator, right? . So shall I reduce the size of the indicator ?.
The minimum dimension allowed is 49 x 28

Reply:
Hi @seanlip , What shall we keep the size of the loading indicator? Because the size of the images may vary from (concerned mainly height) 275px to 28 px.

Reply:
I don't know, sorry. I would ask students or the UI/UX team.

Reply:
Hi @rachelwchen , Could you please help me out. Thanks! 

So the loading indicator is contained in a div with the dimensions supposedly to be same as that of the image. But since we have to display the loading indicator within the div, the size of the loading indicator is a problem here.
The width of the div is fixed so we are mainly concerned with the height of the div and the loading indicator.

Reply:
Hi @vojtechjelinek .